Quantum-Control-Applications/Superconducting/Single Fixed Transmon/Use Case 3 - DRAG calibration/README.md

```diff
@@ -0,0 +1,116 @@
+computational space. The single qubit fidelities were quantified with randomized benchmarking (RB).
+
+## The device
+
+The chip contains 5 2D-transoms, where some of them have nearest-neighbor connectivity, 
+with flux tunable lines. For this experiment, we focus only in one transmon with a drive
+to its flux line to put the transmon in the **sweet-spot**. All other transmons are left
+unbiased leaving them effectively at their zero frequency point.
+
+## Method and results
+
+We will show the determination of the DRAG coefficient with two different pulse sequences.
+The first one is related to the application of `(x180)(-x180)` train of pulses and the second
+is a sequence of `x180y90` and `y180x90` pulses. We are calibrating the $\alpha$ parameter in 
+the following formula,
+
+$\Omega^{'}$ = \Omega + i\alpha\frac{d\Omega/dt}{\Delta}$
+
+Also, based on a protocol listed in Ref. [1] we will calibrate the detuning parameters that
+can lead to a minimized leakage out of the computational state. The pulse sequence is a train
+of `(x180)(-x180)`.
+
+Note that we used `cosine` pulses of `16 ns` in length as the envelope for the control. Note that the
+short pulses (16 ns) bandwidth is `~100 MHz`. This requires mixer calibration to reduce the impact
+of LO leakage on the gate fidelity. The IF frequency was `-95 MHz`.
+
+The experiments below were performed with single shot readout, thus we calibrate
+the discrimination between `|g>` and `|e>` state by looking at the IQ blobs. We also
+needed to rotate the IQ blobs to project the information along one axis only for better
+discrimination.
+
+### DRAG coefficient calibration with `x180` `-x180` pulses
+
+In this step we generate a 2D plot where the x-axis is `number of (x180)(-x180)` pulses
+applied, and the y-axis is the `DRAG coefficient`. By applying a train of pulses, the
+sequence accumulates more and more population in the second excited state with each
+extra pair of `x180` `-x180`, thus we gain sensitivity to pinpoint the DRAG coefficient.
+The pattern generated in the image below resembles a `Chevron-Ramsey`
+experiment. We can identify that the center line of the `Chevron-Ramsey` pattern is away from the zero.
+The figure below the 2D plot shows line cuts at pulse trains with `1`, `6`, and `11` iterations. The minima of these parabolas narrows as the number of iterations in the train of pulses
+increases. We fit quadratic functions to each of these experimental lines and find that for `iterations = 11`,
+the minimum is at `0.081 +/- 0.008`. Note that as the minima gets narrower we need to increase
+the granularity of the data points to accurately capture the experimental behavior. The program corresponding to this sequence can be found
+in `Google_DRAG.py`. The pulse train used for this experiment is a 2D extension of the one listed in Ref. [1].
+
+![Google_DRAg](Google_DRAG.png)
+![Google_linecuts](Google_DRAG_linecuts.png)
```

Comment:
For better readability I would put the two figures next to each other
For that you can use a table 
```
|   2D map    |      Linecuts       |
|:---------------------------:|:-------------------------------:|
| ![chevron](Google_DRAG.png) | ![linecuts](Google_DRAG_linecuts.png)  |
```